src/index.html

```diff
@@ -4,9 +4,360 @@
+            >
+              Laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">
+              tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">
+              photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div class="catalog__wrapper">
+        <div class="catalog__card card" data-qa="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          >
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__stars-and-reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__reviews">Reviews:5</p>
+          </div>
+
+          <div class="card__price-line">
+            <p class="card__product-price">Price:</p>
+
+            <p class="card__product-price-value">$2,199</p>
+          </div>
+
+          <a
+            class="card__buy-reference"
+            href="#"
+            data-qa="card-hover"
+          >
+            BUY
+          </a>
+        </div>
+        <div class="catalog__card card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          >
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__stars-and-reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__reviews">Reviews:5</p>
+          </div>
+
+          <div class="card__price-line">
+            <p class="card__product-price">Price:</p>
+
+            <p class="card__product-price-value">$2,199</p>
+          </div>
+
+          <a class="card__buy-reference" href="#">
+            BUY
```

Comment:
Better write in initial register and in css use [text-transform](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform)